### ISSUE TITLE
What does this syntax mean and do for SSH "<volume-mounts>"

### ISSUE THREAD

**Issue by janat08** [codenvy/codenvy#2082]
Additionaly the doc's links are all broken for SSH https://codenvy.io/docs/user-guide/ssh/index.html#public--private-key-generation .

**Comment by janat08** [comment]
I suggest you try and make GUI port for people who're really trying to save time by like requiring of them to only launch che from docker after which comes ssh and file sync.

**Comment by janat08** [comment]
-v ~/Docker/:/data is what volume mounts wants. Other doc topics ignore symantics and just suggest that you put a -v <path>:/data .

**Comment by janat08** [comment]
Still won't work.

**Comment by janat08** [comment]
I think I'm doing the right stuff, but it times out connecting to 172.17.0.1:80. Once I got message about dns problems but my laptop is already configured to use 8.8.8.8

**Comment by eivantsov** [comment]
@janat08 thanks for pointing to a dead link.

As to your question, I am a little bit confused on what problem you are trying to solve here. Can you summarize it?

**Comment by janat08** [comment]
I point out that command instructions are non standard on that link. I can't ssh into my workspace as it just times out.

**Comment by janat08** [comment]
Left out -url parameter that I thought would be optional
